src/features/resources/ResourceCardActions.tsx

```diff
@@ -8,6 +8,10 @@ import { useTranslation } from "react-i18next";
 import SyntaxHighlighter from "react-syntax-highlighter";
 import { monokai } from "react-syntax-highlighter/dist/esm/styles/hljs";
 
+import PDFDialogButton from "common/components/PDFDialogButton";
+
+const PROXY_URL = "https://cors-anywhere.herokuapp.com/";
```

Comment:
I don't really want to merge this but I guess this is required until we solve the CORS issue.
If we must merge this, please add an issue to timeline-patient to make us remember to fix this

Reply:
Well this line is removed in the next PR on minIO config